pkg/wasm/imagefetcher.go

```diff
@@ -61,6 +65,15 @@ func NewImageFetcher(ctx context.Context, opt ImageFetcherOption) *ImageFetcher
 	} else {
 		fetchOpts = append(fetchOpts, remote.WithAuth(&authn.Basic{Username: opt.Username}))
 	}
+
+	if opt.Insecure {
+		t := remote.DefaultTransport.Clone()
+		t.TLSClientConfig = &tls.Config{
+			InsecureSkipVerify: opt.Insecure, //nolint: gosec
```

Comment:
What about plaintext repos?

Reply:
seems some error for now
```console
2022-01-26T02:48:54.028581Z	error	wasm	cannot fetch Wasm module oci://kind-registry:5000/zirain/autocomplete-authority:v1: could not fetch Wasm OCI image: could not fetch image: Get "https://kind-registry:5000/v2/": http: server gave HTTP response to HTTPS client

```

Reply:
I didn't look at the code but I would note that Helm just added OCI support and plaintext works out of the box.

But it's possible it read some local config I have on my machine and I had configure the registry as plaintext... not sure

Reply:
I think we should support plaintext repo, especially local develop with kind.

Reply:
Yeah sounds good to allow pull from plain text by default.